src/test/spec/unified_runner/mod.rs

```diff
@@ -31,14 +32,21 @@ pub use self::{
     test_runner::{EntityMap, TestRunner},
 };
 
+use super::run_spec_test_with_path;
+
 static MIN_SPEC_VERSION: Version = Version::new(1, 0, 0);
-static MAX_SPEC_VERSION: Version = Version::new(1, 7, 0);
+static MAX_SPEC_VERSION: Version = Version::new(1, 10, 0);
+
+fn file_level_log(message: impl AsRef<str>) {
```

Comment:
This function is used to log stuff in a way that separates it from other files. I took the format from some stuff I wrote in the Swift runner a long time ago, let me know if you think it looks good.

Here's a sample:
```
cargo test sessions::run_unified
   Compiling mongodb v2.4.0 (/home/patrick/mongo-rust-driver)
    Finished test [unoptimized + debuginfo] target(s) in 53.69s
     Running unittests (target/debug/deps/mongodb-dc9092005b6c81b2)

running 1 test

------------
Running tests from /home/patrick/mongo-rust-driver/src/test/spec/json/sessions/snapshot-sessions-unsupported-ops.json

Executing "Server returns an error on insertOne with snapshot"
Executing "Server returns an error on insertMany with snapshot"
Executing "Server returns an error on deleteOne with snapshot"
Executing "Server returns an error on updateOne with snapshot"
Executing "Server returns an error on findOneAndUpdate with snapshot"
Executing "Server returns an error on listDatabases with snapshot"
Executing "Server returns an error on listCollections with snapshot"
Executing "Server returns an error on listIndexes with snapshot"
Executing "Server returns an error on runCommand with snapshot"

------------
Skipping file /home/patrick/mongo-rust-driver/src/test/spec/json/sessions/snapshot-sessions-not-supported-client-error.json: client topology not compatible with test


------------
Running tests from /home/patrick/mongo-rust-driver/src/test/spec/json/sessions/driver-sessions-dirty-session-errors.json

Executing "Dirty explicit session is discarded (insert)"
Executing "Dirty explicit session is discarded (findAndModify)"
Executing "Dirty implicit session is discarded (insert)"
Executing "Dirty implicit session is discarded (findAndModify)"
Executing "Dirty implicit session is discarded (read returning cursor)"
Executing "Dirty implicit session is discarded (read not returning cursor)"

------------
Skipping file /home/patrick/mongo-rust-driver/src/test/spec/json/sessions/snapshot-sessions-not-supported-server-error.json: client topology not compatible with test


------------
Running tests from /home/patrick/mongo-rust-driver/src/test/spec/json/sessions/driver-sessions-server-support.json

Executing "Server supports explicit sessions"
Executing "Server supports implicit sessions"

------------
Running tests from /home/patrick/mongo-rust-driver/src/test/spec/json/sessions/snapshot-sessions.json

Executing "Find operation with snapshot"
Executing "Distinct operation with snapshot"
Executing "Aggregate operation with snapshot"
Executing "countDocuments operation with snapshot"
Executing "Mixed operation with snapshot"
Executing "Write commands with snapshot session do not affect snapshot reads"
Executing "First snapshot read does not send atClusterTime"
Executing "StartTransaction fails in snapshot session"
test test::spec::sessions::run_unified ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 391 filtered out; finished in 10.90s
```